pkg/ruler/memstore.go

```diff
@@ -46,23 +46,23 @@ func ForStateMetric(base labels.Labels, alertName string) labels.Labels {
 	return b.Labels()
 }
 
-type Metrics struct {
-	Evaluations *prometheus.CounterVec
-	Samples     prometheus.Gauge       // in memory samples
-	CacheHits   *prometheus.CounterVec // cache hits on in memory samples
+type memstoreMetrics struct {
```

Comment:
I don't think we need to make these private, but I _did_ link you to `ingesterMetrics` as an example to follow which _are_ private though :laughing:. In the long term, I think we should expose them consistently and I'd favor public Metric structs where possible.